keras_retinanet/preprocessing/open_images.py

```diff
@@ -0,0 +1,206 @@
+import csv
+import json
+import os
+
+import numpy as np
+from PIL import Image
+
+from .generator import Generator
+from ..utils.image import read_image_bgr
+
+
+def get_labels(metadata_dir):
+    trainable_classes_path = os.path.join(metadata_dir, 'classes-bbox-trainable.txt')
+    description_path = os.path.join(metadata_dir, 'class-descriptions.csv')
+
+    description_table = {}
+    with open(description_path) as f:
+        for row in csv.reader(f):
+            if len(row):
```

Comment:
Are there also empty rows? Otherwise I'd remove this.

Reply:
The last line in the class-descriptions csv is empty

Reply:
i'll add some commentary in the code